integration-cli/docker_cli_attach_unix_test.go

```diff
@@ -170,7 +170,7 @@ func (s *DockerSuite) TestAttachDetach(c *check.C) {
 
 	select {
 	case <-ch:
-	case <-time.After(10 * time.Millisecond):
+	case <-time.After(100 * time.Millisecond):
```

Comment:
Ah, I remember that I tweak the timeout value to avoid some false positive test failure, but some concerns raised to worry about this behavior probably paper cover some potential issue, e.g, performance 😄 .

Reply:
This is interesting. I checked git history, the only meaningful change to this test was done in commit ae0883ce009daa2b94e67264cfa5a13b4cae76bf when it was moved to `integration-cli`.

The original test (before the above commit) was 
* ...
* sending escape sequence to "attach";
* checking that "attach" is exiting (after an escape sequence is sent);
* checking that the container is still alive;
* kills the containers.
Also, timeouts were big at that time :)

The new test (after the commit):
* sends the escape sequence to "attach";
* checks that the container is running;
* kills the container;
* checks that the "attach" has exited.

It seems that the order is very broken. I'll send another PR.